scaffolding-ruby/lib/scaffolding.sh

```diff
@@ -196,18 +196,24 @@ scaffolding_bundle_install() {
 
   build_line "Installing dependencies using $(_bundle --version)"
   start_sec="$SECONDS"
-  _bundle_install \
-    "$CACHE_PATH/vendor/bundle" \
-    --retry 5
+
+  {
+    _bundle_install \
+      "$CACHE_PATH/vendor/bundle" \
+      --retry 5
+  } || {
```

Comment:
Nice!!! This is what we need

Reply:
![oooooo](https://user-images.githubusercontent.com/517302/29937031-3664c7ce-8e52-11e7-97f4-ab5a78960558.gif)